extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQWorkerTaskLauncher.java

```diff
@@ -150,7 +156,8 @@ public MSQWorkerTaskLauncher(
       final OverlordClient overlordClient,
       final WorkerFailureListener workerFailureListener,
       final Map<String, Object> taskContextOverrides,
-      final long maxTaskStartDelayMillis
+      long maxTaskStartDelayMillis,
```

Comment:
style: let's not mix `final` and non-`final` in the same argument list unless the non-`final` things really need to be reassignable. Here, they don't.

My personal preference is that the things are all marked `final`. (I tend to use `final` a lot because I wish `final` was the default in Java, and I wish a keyword was needed to make something not-`final`. The closest I can get to that is marking everything `final`.)